src/mpi/init/init_util.c

```diff
@@ -69,7 +71,14 @@ void MPII_Call_finalize_callbacks(int min_prio, int max_prio)
     }
 }
 
-static const char *threadlevel_name(int threadlevel)
+/**
+ * @brief   Convert thread level integer to a string representation
+ *
+ * @param   threadlevel Thread level integer (input)
+ *
+ * @return  char*       String representation of thread level
+ */
+const char *threadlevel_name(int threadlevel)
```

Comment:
The convention is to add prefix for non-static functions. How about `MPII_threadlevel_name`?

Reply:
Sure. Renamed the function as you suggest.